modular_skyrat/modules/encounters/code/nri_raiders.dm

```diff
@@ -1,3 +1,35 @@
+/// Possible places/departments to inspect
+#define INSPECTION_LIST pick("Cargo",\
+							"Command",\
+							"Engineering",\
+							"Medical",\
+							"Science",\
+							"Service",\
+							"Security")
+
+/// List of possible decorative job titles for additional roleplay potential
+#define NRI_JOB_LIST pick("NRI Patrol Officer",\
+					"NRI Satellite-Patrol Officer",\
+					"NRI Yefreitor-Patrol Officer",\
+					"NRI Junior Patrol Officer",\
+					"NRI Government Investigator",\
+					"NRI Field Inspector",\
+					"NRI Counter-Truancy Officer",\
+					)
+
+/// List of possible decorative leader job titles for additional roleplay potential
+#define NRI_LEADER_JOB_LIST pick("NRI Lieutenant Officer",\
+					"NRI Sergeant Officer",\
+					"NRI Senior Patrol Officer",\
+					"NRI Lead Investigator",\
+					"NRI Lead Inspector",\
+					)
+
+/// Amount of items to "confiscate", lower amount
+#define CONFISCATE_LOWER 10
+/// Amount of items to "confiscate", higher amount
+#define CONFISCATE_HIGHER 25
```

Comment:
All of the defines here should have a matching `#undef DEFINE_NAME_HERE` at the end of the file, to not pollute the global namespace.